docker-compose.yml

```
@@ -72,6 +72,21 @@ services:
     depends_on:
       - vault
 
+  product_incentory_db:
+    image: postgres:13-alpine
+    container_name: ecomm-product_inventory-db
```

The container name was fine. I was talking about the service name in the previous comment.

```suggestion
  product_inventory_db:
    image: postgres:13-alpine
    container_name: ecomm-product-db
```